src/scalars/GraphQLPositiveFloat.js

```diff
@@ -8,7 +8,7 @@ function convert(value) {
         throwNegativeValueError(value, POSITIVE_FLOAT);
     }
 
-    return value + '';
+    return normalizeFloatingPointNumbers(value + '');
```

Comment:
Isn't it better to pass the value as is and do the string conversion in the utility function itself?

Reply:
Actually yes, good point. I'll update the code.